numpy/lib/npyio.py

```diff
@@ -1610,7 +1613,8 @@ def genfromtxt(fname, dtype=float, comments='#', delimiter=None,
         for example, `file` would become `file_`.
     deletechars : str, optional
         A string combining invalid characters that must be deleted from the
-        names.
+        names. In case `comments` is something other than '#', this must be
+        an empty string in order to preserve special characters in the names.
```

Comment:
What is the default value and why does it need to be empty in the case that `#` is not the character that prefixes comments. Not saying that this is wrong, but that sounds like strange behavior, maybe a bug.

Reply:
See [#13088(comment)](https://github.com/numpy/numpy/issues/13088#issuecomment-470006776), in case `#` is not the character that prefixes comments and something else is used, then `#` disappears from the header name, yes it's probably a bug, but I'm documenting it until it gets fixed at least.